contracts/Core/interface/ICollectionManager.sol

```diff
@@ -81,4 +81,9 @@ interface ICollectionManager {
      * @return epoch in which the registry needs to be updated
      */
     function getUpdateRegistryEpoch() external view returns (uint32);
+
+    /**
+     * @return current depth of the merkle tree
+     */
+    function getDepth() external view returns (uint256);
```

Comment:
This is not required. Can be removed from the interface

Reply:
Done @SamAg19